modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -2066,8 +2085,38 @@ class ConvolutionLayerImpl CV_FINAL : public BaseConvolutionLayerImpl
         {
             int nstripes = std::max(getNumThreads(), 1);
 
+            if (!variableWeight)
```

Comment:
Why do you need variableWeight check?

Reply:
I am concerned that the weights are initialized in the `forward` every time when the variable is entered, which will make the model run slower, especially for winograd.

Reply:
In the latest code, I still reserve the `variableWeight ` check, because we need to initialize each time when the weight is variable.